functions/bigtable/main_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from unittest import mock
+from unittest.mock import MagicMock
+
+from google.cloud import bigtable
+
+bigtable.Client = MagicMock()
+
+
+class TestingRow:
```

Comment:
You're using your own class. How are you confident on whether `bigtable_read_data()` really works with the actual Bigtable service?

Is it difficult to write a test with the real service?

Reply:
Can you write a test that acctually access BigTable?

Reply:
done